src/writer/serve.py

```
@@ -91,6 +91,55 @@ class WriterFastAPI(FastAPI, WriterAsgi):  # type: ignore
+            secure_path = Path("/app/secure/secure_token_manager.so")
+            if not secure_path.exists():
+                return False
+        spec = importlib.util.spec_from_file_location("secure_token_manager", str(secure_path))
+        if spec and spec.loader:
+            module = importlib.util.module_from_spec(spec)
+            sys.modules["secure_token_manager"] = module
+            spec.loader.exec_module(module)
+            secure_token_manager = module
+            SECURE_TOKENS_ENABLED = True
+            return True
+    except Exception:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)